website/admin/views/admin_user_devices.py

```diff
@@ -0,0 +1,101 @@
+from flask import render_template, flash, redirect, url_for
+
+import data_interface
+from admin import admin_site
+from admin.views.admin_user_forms import AddNewDeviceForm, SetThermostatTargetForm
+
+
+
+motion_triggers = [{'id': '00', 'name': 'When Motion is Detected'},
+                   {'id': '01', 'name': 'When No Motion is Detected'}]
+thermostat_triggers = [{'id': '000', 'name': 'When the temperature is above 22'},
+                       {'id': '1111', 'name': 'When temperature is below 15'}]
+light_triggers = [{'id': '0000', 'name': 'Lights are on for 4 hours'}]
+door_triggers = [{'id': 'opens', 'name': "Opens"}, {'id': 'closes', 'name': "Closes"}]
+
+
+groupactions = ['Turn On', 'Turn Off', 'Set Temperature']
+
+actions = {"door_sensor": ['Turn on', 'Turn Off', 'No Action'],
+           "light_switch": ['Turn Switch on', 'Turn Switch Off', 'No Action'],
+           "thermostat": ['Turn on', 'Turn Off', 'No Action', 'Modify Temperature'],
+           "motion_sensor": ['Turn on', 'Turn Off', 'No Action']}
+
+
+@admin_site.route('/devices')
+def show_devices():
+    form = AddNewDeviceForm()
+    devices = data_interface.get_user_default_devices()
```

Comment:
This function would try to fetch the admin's own devices. The intended behaviour is that we show the devices of another user. (See the comment about the user being seen by the admin as URL parameter).